build/Targets/Packages.props

```diff
@@ -89,6 +89,7 @@
     <MicrosoftVisualStudioProgressionCodeSchemaVersion>15.0.26507-alpha</MicrosoftVisualStudioProgressionCodeSchemaVersion>
     <MicrosoftVisualStudioProgressionCommonVersion>15.0.26507-alpha</MicrosoftVisualStudioProgressionCommonVersion>
     <MicrosoftVisualStudioProgressionInterfacesVersion>15.0.26507-alpha</MicrosoftVisualStudioProgressionInterfacesVersion>
+    <MicrosoftVisualStudioProjectSystemVersion>15.0.751</MicrosoftVisualStudioProjectSystemVersion>
```

Comment:
Needs to be 15.3.178-pre-g209fb07c2e or later so you get the appropriate System.Composition dependency.